lib/Site.js

```diff
@@ -541,6 +541,20 @@ Site.prototype.generatePages = function () {
   });
 };
 
+/**
+ * Checks if variables file has been changed and rebuild variables map when necessary
```

Comment:
- Phrase it similar to the function that it wraps:
  "Recollects the user defined variables map in the site/subsites if needed"

- Place it just under the function that it wraps.